zqd/ingest/legacypcap.go

```diff
@@ -0,0 +1,290 @@
+package ingest
+
```

Comment:
@mattnibs : is this file just a copy of the pre-existing pcap ingest code? The diffs make it challenging to know what to review here & in the new archive specific pcap work.

Reply:
For the most part. I'll keep the rename legacypcap.go -> pcap.go and give the current pcap.go another name, to make this pr more reviewable. After this merges I can file a followup pr to change the names back.

Reply:
Done.